cinema_service/settings.py

```diff
@@ -47,6 +47,7 @@
     "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in the `INSTALLED_APPS`. Ensure it is only enabled in development environments to avoid performance issues and potential security risks in production.